sprints/dashboard/models.py

```diff
@@ -124,12 +124,36 @@ def assignee_time(self) -> int:
 
         return max(self.time_estimate - self.review_time, 0)  # We don't want negative values here.
 
+    def get_review_time_from_story_points(self):
+        issue_story_points = self.story_points
+        review_hours_by_story_points = settings.SPRINT_HOURS_RESERVED_FOR_REVIEW
+        if issue_story_points is None:
+            return review_hours_by_story_points.get("null")
```

Comment:
Or something along these lines.
```suggestion
        if (hours := review_hours_by_story_points.get(issue_story_points)) is not None:
            return hours
```